FourthTask/sorting/construct/construct.go

```diff
@@ -0,0 +1,63 @@
+//it works, but i have one problem there
+package construct
+
+import (
+	"fmt"
+
+	"sync"
+
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/sorting/types"
+)
+
+var wg sync.WaitGroup
+
+func People() (res types.People) { //i'll left some questions
+	exit := make(chan string) //this chan i'll use to exit function
```

Comment:
Seems to be an overkill to use chans/goroutines here.

Reply:
i think, that i woun't to change this file(only if i will have some time). in next task i will use goroutines and it will be really usefull. here i will just make from this goruitines simple functions.

Reply:
As we are mentioned over engineering it's not a good thing, as well it's hard to as to review a lot of creative ideas where people just want to bring some technology to task just for fun

Reply:
Please re-implement it without go-routines.